### DISCUSSION TITLE
Osc mode range changing pins.

### DISCUSSION THREAD

**Discussion by santlaha** [fhdm-dev/scoppy#31] [Q&A]
What are the switching pins for osc mode.?

**Comment by fhdm-dev** [comment]
> What are the switching pins for osc mode.?

GPIO 2 & 3 for channel 1, GPIO 4 & 5 for channel 2.
See [here](https://github.com/fhdm-dev/scoppy/wiki/Analog-Front-End) for details of how the pins map to the different voltage ranges.
